templates/prom-alertmanager.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>prom-alertmanager</Name>
+  <Repository>prom/alertmanager:latest</Repository>
+  <Registry>https://hub.docker.com/r/prom/alertmanager</Registry>
+  <Network>bridge</Network>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support>https://forums.unraid.net/topic/87798-support-selfhostersnets-template-repository/</Support>
+  <Project>https://github.com/prometheus/alertmanager</Project>
+  <Overview>The Alertmanager handles alerts sent by client applications such as the Prometheus
+    server. &#xD;
+    It takes care of deduplicating, grouping, and routing them to the correct receiver integrations
+    such as email, PagerDuty, OpsGenie, or many other mechanisms thanks to the webhook receiver.
+    &#xD;
+    It also takes care of silencing and inhibition of alerts.&#xD;
+    &#xD;
+    alertmanager/alertmanager.yml&#xD;
+    This configuration contains information about which channels to send to. For simplicity, we use
+    e-mail. Refer to the Alertmanager docs to learn about other channels.&#xD;
+    &#xD;
+    To configure prometheus to use these alerts, add the below to prometheus/prometheus.yml:&#xD;
+    alerting:&#xD;
+    alertmanagers:&#xD;
+    - scheme: http&#xD;
+    static_configs:&#xD;
+    - targets: &amp;lt; 'alertmanager:9093' &amp;gt;</Overview>
+  <Category>Productivity:</Category>
+  <WebUI>http://[IP]:[PORT:9093]</WebUI>
+  <TemplateURL>
+    https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/prom-alertmanager.xml</TemplateURL>
+  <Icon>
+    https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/img/prometheus.png</Icon>
+  <Requires />
+  <Config Name="Port" Target="9093" Default="9093" Mode="tcp" Description="" Type="Port"
+    Display="always" Required="false" Mask="false">9093</Config>
+  <Config Name="ConfigFile" Target="/etc/alertmanager/alertmanager.yaml"
+    Default="/mnt/user/appdata/alertmanager/etc/alertmanager/alertmanager.yaml" Mode="rw"
+    Description="Config directory" Type="Path" Display="always" Required="false" Mask="false">
+    /mnt/user/appdata/alertmanager/etc/alertmanager/alertmanager.yaml</Config>
+  <Config Name="TemplateURL" Target="TemplateURL"
+    Default="https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/prom-alertmanager.xml"
+    Mode="" Description="" Type="Label" Display="advanced-hide" Required="false" Mask="false">
+    https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/prom-alertmanager.xml</Config>
+  <Config Name="Appdata" Target="" Default="/mnt/user/appdata/alertmanager" Mode="rw"
```

Comment:
We'll use `/mnt/user/app data/alertmanager` as the root directory on Unraid to store all things related to the container, although this path doesn't directly map to anything inside the container. So this `<Config>` element is correct, good job!

Let's add a second path that maps `/mnt/user/app data/alertmanager/storage` to [`/alertmanager`](https://github.com/prometheus/alertmanager/blob/c920b605b647f2548e23a3dc2fea014b9c9dbf43/Dockerfile#L17) inside the container

Reply:
Fixed